package.json

```diff
@@ -35,33 +35,28 @@
     "url": "http://www.bentley.com"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "~5.55.0",
-    "@typescript-eslint/parser": "~5.55.0",
-    "eslint-import-resolver-node": "0.3.4",
```

Comment:
are you sure about removing these import resolvers?

Reply:
https://www.npmjs.com/package/eslint-import-resolver-node
https://www.npmjs.com/package/eslint-import-resolver-typescript

Reply:
is eslint-plugin-import even used anymore? I think all its rules are disabled so we might as well remove the dependency

Reply:
anmol plz verify and upate accordingly

Reply:
I'm not sure what you're looking for, I don't think the plugin can be used that way. Usually it's `import/<rule-name-here>`. But with a flat config, just look for the full package name to see if it's imported into any configs. I can see it's used in two configs, but none of its rules are enabled so it's just dead weight.

Reply:
Yes
![image](https://github.com/iTwin/eslint-plugin/assets/98850418/aa682717-2a74-46ca-beed-f448a1690b04)

![image](https://github.com/iTwin/eslint-plugin/assets/98850418/b4cb6867-123e-458d-8556-f2fe72605d92)

Reply:
I was talking about `eslint-plugin-import`, I guess it might use one of those resolvers under the hood... if you're going to remove the resolvers, you should remove that plugin too